src/UserActivity.js

```diff
@@ -110,13 +111,14 @@ class UserActivity {
 
     _onPageVisibilityChanged(e) {
         if (document.visibilityState === "hidden") {
+            console.log("page hidden");
```

Comment:
Not sure if meant to keep this in the commit or not... Either way is fine, just wanted to check.